File: EticaretJava/app/src/main/java/com/example/eticaretjava/model/Payment.java
package com.example.eticaretjava.model;

import com.google.gson.annotations.SerializedName;

public class Payment {

    public static class PaymentDto {
        @SerializedName("provider")
        public String provider;

        @SerializedName("status")
        public String status;

        @SerializedName("amount")
        public double amount;

        @SerializedName("created_at")
        public String createdAt;
    }
}

File: Eticaretjetpack/app/src/main/java/com/example/eticaretjetpack/model/Payment.kt
package com.example.eticaretjetpack.model

import com.google.gson.annotations.SerializedName

data class PaymentDto(
    @SerializedName("provider")
    val provider: String,

    @SerializedName("status")
    val status: String,

    @SerializedName("amount")
    val amount: Double,

    @SerializedName("created_at")
    val createdAt: String
)
